circuit_knitting/cutting/cutqc/dynamic_definition.py

```diff
@@ -150,6 +157,12 @@ def dd_build(
     return dd_bins
 
 
+@deprecate_func(
+    removal_timeline="Circuit knitting toolbox 0.8.0 release",
+    since="0.7.0",
+    package_name="circuit-knitting-toolbox",
+    additional_msg="Use the wire cutting or automated cut-finding functionality in the `circuit_knitting.cutting` package. ",
```

Comment:
backticks

Reply:
I won't keep listing them, but I think you can remove them all since they're not being rendered anywhere but stdout